docs/rules/terraform_required_providers.md

```diff
@@ -10,7 +10,7 @@ rule "terraform_required_providers" {
 }
 ```
 
-## Example
+## Examples
 
 ```hcl
 provider "template" {}
```

Comment:
The issue message has been updated, so it would be nice to reflect it in the example output as well.